File: src/main/java/architecture/goldenboughs_lib/mixed/client/IKeyMapping.java
package architecture.goldenboughs_lib.mixed.client;

import architecture.goldenboughs_lib.api.NoMixinException;
import net.minecraft.client.KeyMapping;

public interface IKeyMapping {
	static IKeyMapping of(KeyMapping obj) {
		return obj;
	}

	default int goldenboughs_lib$getClickCount() {
		throw new NoMixinException();
	}
}

File: src/main/kotlin/architecture/goldenboughs_lib/mixed/client/IKeyMapping.kt
package architecture.goldenboughs_lib.mixed.client

import architecture.goldenboughs_lib.api.NoMixinException
import net.minecraft.client.KeyMapping

interface IKeyMapping {
	fun `goldenboughs_lib$getClickCount`(): Int {
		throw NoMixinException()
	}

	companion object {
		@JvmStatic
		fun of(obj: KeyMapping): IKeyMapping {
			return obj
		}
	}
}
